plots/gender_by_language.py

```diff
@@ -7,23 +7,23 @@
 from bokeh.resources import CDN
 from bokeh.embed import autoload_static
 import os
-import pandas
+import pandas as pd
 from numpy import max, min
 
 
 def plot(newest_changes):
-    filelist = os.listdir('/home/maximilianklein/snapshot_data/{}/'.format(newest_changes))
+    filelist = os.listdir('/home/vivekrai/snapshot_data/{}/'.format(newest_changes))
```

Comment:
@hargup I have that open already in another issue #13. But this is what we have for the time being. I'll change the path when merging.